x/launch/keeper/request.go

```diff
@@ -133,7 +133,7 @@ func checkAccount(ctx sdk.Context, k Keeper, chainID, address string) (bool, err
 
 // applyRequest approves the request and performs
 // the launch information changes
-func applyRequest(
+func ApplyRequest(
```

Comment:
I like the idea of a separete package for tests, but only for black-box tests, because if we do it for everything, we need to export (make public) all methods. It can be harder in the future when the application starts to grow a lot.

Reply:
Yes but I think most methods can be made public in the keeper. For `ApplyRequest ` it totally makes sense in my mind since this is an action that could be performed by another module in theory.

Internal methods should be used as helper, and if there are few we can cover them by testing the high-level methods

Reply:
Also using the same package would require having the method returning a keeper instance for testing purpose in the same package, I'm not a big fan of this idea